.github/workflows/config-jsonschema-check.yml

```diff
@@ -0,0 +1,123 @@
+  group: ${{ (
+    github.ref == 'refs/heads/master' ||
+    startsWith(github.ref, 'refs/heads/release/') ||
+    startsWith(github.ref, 'refs/tags/')) &&
+    format('{0}-{1}', github.run_id, github.run_attempt) ||
+    format('{0}-{1}-release', github.workflow, github.ref) }}
+  cancel-in-progress: true
+
+jobs:
+  jsonschema-check:
+    # Run on push to the 'master' and release branches of tarantool/tarantool
+    # or on pull request if the 'notest' label is not set.
+    if: github.repository_owner == 'tarantool' &&
+        ( github.event_name != 'pull_request' ||
+          ( github.event_name == 'pull_request' &&
+            !contains(github.event.pull_request.labels.*.name, 'notest') ) )
+
+    runs-on: ubuntu-20.04-self-hosted
+
+    timeout-minutes: 60
+
+    container:
+      image: docker.io/tarantool/testing:ubuntu-focal
+      # Mount /dev to the container to be able to mount a disk image inside it
+      # for successful run of the .github/actions/environment action.
+      volumes:
+        - /dev:/dev
+      # Our testing expects that the init process (PID 1) will
+      # reap orphan processes. At least the following test leans
+      # on it: app-tap/gh-4983-tnt-e-assert-false-hangs.test.lua.
+      # Add extra privileges to the container for successful run
+      # of the .github/actions/environment action.
+      options: '--init --privileged'
+
+    steps:
+      - name: Prepare checkout
+        uses: tarantool/actions/prepare-checkout@master
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: recursive
+          repository: tarantool/tarantool
+          ref: ${{ inputs.submodule && 'master' || github.ref }}
+      - uses: ./.github/actions/environment
+      - name: Install deps
+        uses: ./.github/actions/install-deps-debian
+      - name: Optional submodule bump
+        if: ${{ inputs.submodule }}
+        uses: ./.github/actions/bump-submodule
+        with:
+          submodule: ${{ inputs.submodule }}
+          revision: ${{ inputs.revision }}
+      - name: Build
+        run: make -f .test.mk build
+
+      - name: Install jq, check-jsonschema
+        run: |
+          apt-get update
+          apt-get install jq python3-pip -y
+          pip3 install check-jsonschema
+      - name: Generate config schema
+        run: |
+          ./src/tarantool -e "print(require('json').encode(require('config'):jsonschema()))" > config-schema.json
+      - name: Pretty format the schema
+        run: jq --sort-keys -M . config-schema.json > config-schema-pretty.json
+      - name: Validate schema against metaschema
+        run: |
+          validation_output=$(check-jsonschema --check-metaschema config-schema-pretty.json)
+          if echo "$validation_output" | grep -q "ok -- validation done"; then
+            echo "Schema validation passed"
+          else
+            echo "Schema validation failed"
+            exit 1
+          fi
```

Comment:
As I see, `check-jsonschema` sets exit code 1 at exit if there is an error. So, I guess, we can just run it without checking the output.

Reply:
output checks removed.